src/project/FileTreeView.js

```diff
@@ -478,7 +478,42 @@ define(function (require, exports, module) {
         render: function () {
             var fullname = this.props.name,
                 extension = LanguageManager.getCompoundFileExtension(fullname),
-                name = _getName(fullname, extension);
+                name = _getName(fullname, extension),
+                fileType = "default";
+
+            switch (extension.toLowerCase()) {
+              case 'htmls':
```

Comment:
OK, I read it incorrectly, it returns it *without* the `.` so this is good, see https://github.com/mozilla/brackets/blob/master/src/language/LanguageManager.js#L401